cmake/branding/generate_icons_linux.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Recolor icon
+echo Recoloring vector icon...
+case "$1" in
+	"-r" ) # Release
+	sed 's/#780116/#249a56/g; s/#c51306/#50d99b/g' icon.svg > recolored.svg
+	sed 's/#780116/#249a56/g; s/#c51306/#50d99b/g' splash.svg > splash-recolored.svg
+	;;
+	"-b" | "-rc" ) # Beta/Release Candidate
+	sed 's/#780116/#0267c1/g; s/#c51306/#09a9d9/g' icon.svg > recolored.svg
+	sed 's/#780116/#0267c1/g; s/#c51306/#09a9d9/g' splash.svg > splash-recolored.svg
+	;;
+	"-c" | "-n" ) # Canary/Nightly
+	sed 's/#780116/#ffa40f/g; s/#c51306/#fad200/g' icon.svg > recolored.svg
+	sed 's/#780116/#ffa40f/g; s/#c51306/#fad200/g' splash.svg > splash-recolored.svg
+	;;
+	* ) # No type specified, leave it be
+	sed '' icon.svg > recolored.svg
+	sed '' splash.svg > splash-recolored.svg
+	;;
+esac
+
+# Generate raster images
+echo Generating raster images for icon...
+for RES in 16 24 32 48 64 96 128
+do
+	mkdir -p icons/${RES}x${RES}/apps
+	rsvg-convert -w $RES -h $RES -o "icons/${RES}x${RES}/apps/lmms.png" recolored.svg
```

Comment:
Lexicologically, I'd recommend `recolored.svg -o lmms.png` instead of `-o lmms.png recolored.svg` as it's easier to read (same applies for all other instances of this command).

Reply:
Changed 👍